lib/semantics/CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ add_library(FortranSemantics
   check-return.cc
   check-stop.cc
   expression.cc
+  io.cc
```

Comment:
`check-io.cc` would be more consistent with the other file names.

Reply:
Done.